test/numbers.jl

```diff
@@ -830,6 +830,47 @@ end
     @test cmp(isless, 1, NaN) == -1
     @test cmp(isless, NaN, NaN) == 0
 end
+@testset "ispositive/isnegative" begin
```

Comment:
It'd be better to test in a for loop multiple types (e.g. all `IEEEFloat`, and similarly for the standard integers), instead of testing just two types.